src/tabs/js/jquery.tabs.js

```diff
@@ -0,0 +1,67 @@
+/* ==========================================================================
+   Tabs
+
+   Initialize:
+   cargobay.tabs.init();
+
+   Support:
+   Latest Chrome
+   Latest FireFox
+   Latest Safari
+   IE9 and up
+   ========================================================================== */
+
+
+var cargobay = cargobay || {};
+
+cargobay.tabs = (function($, window, undefined) {
+
+    var init, activateTabs, updatePanes;
+
+    // Config
+    var tabClass = 'js-tab',
+        tabClassActive = 'tab--active',
+        paneClassActive = 'tab-pane--active';
+
+
+    // Init
+    init = function() {
```

Comment:
This is not necessary, you can just return the `activateTabs` as init on line 63.

````
return {
    init: activateTabs
};
````